bouncer.go

```diff
@@ -309,8 +316,18 @@ type Login struct {
 	Expire string `json:"expire"`
 }
 
-func handleErrorServeHTTP(bouncer *Bouncer, remoteIP, remediation string, rw http.ResponseWriter, req *http.Request) {
-	bouncer.log.Debug(fmt.Sprintf("handleErrorServeHTTP ip:%s remediation:%s", remoteIP, remediation))
+func handleBanServeHTTP(bouncer *Bouncer, rw http.ResponseWriter) {
+	rw.WriteHeader(http.StatusForbidden)
+	if bouncer.banTemplate != nil {
+		err := bouncer.banTemplate.Execute(rw, map[string]string{"caca": "caca"})
```

Comment:
:hankey:

Reply:
@maxlerebourg

Reply:
See https://github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin/pull/145 which revert to previous state.
I kept the new name for the ban function